packages/kv-asset-handler/src/test/getAssetFromKV.ts

```diff
@@ -1,134 +1,122 @@
-import test from "ava";
```

Comment:
is ava -> vitest what was not supported on Node 22?
Maybe call the change out in the PR description?